backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/geocoding/query.kt

```diff
@@ -0,0 +1,72 @@
+package app.ehrenamtskarte.backend.stores.geocoding
+
+import app.ehrenamtskarte.backend.common.COUNTRY_CODE
+import app.ehrenamtskarte.backend.common.STATE
+import app.ehrenamtskarte.backend.stores.importer.types.AcceptingStore
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.ktor.client.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import org.geojson.Feature
+import org.geojson.FeatureCollection
+import java.io.File
+
+suspend fun queryFeatures(params: List<Pair<String, String>>): List<Feature> {
```

Comment:
For testability and also cleanness I think It would be create to have a class like `FeatureFetcher` with those functions. Then an object can be passed to the Sanitize step. 

If this becomes a proper class then we can also reuse the `HttpClient` by adding it as an instance field to the `FeatureFetcher` class. Especially when we use the caching functionality of the HttpClient, then it should be initialized in the `DataImporter`.

Reply:
Done